showcase/src/app/demo/app-data/demo-app-data.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit, Inject } from "@angular/core";
+import { ReferenceLink } from "../../shared/reference-block";
+import {
+	STARK_USER_SERVICE,
+	StarkUserService,
+	STARK_LOGGING_SERVICE,
+	StarkLoggingService,
+	StarkUser,
+	StarkApplicationMetadata,
+	StarkSessionService,
+	STARK_SESSION_SERVICE,
+	STARK_APP_METADATA
+} from "@nationalbankbelgium/stark-core";
+import * as moment from "moment";
+const componentName: string = "demo-app-data";
+
+@Component({
+	selector: "demo-app-data",
+	templateUrl: "./demo-app-data.component.html"
+})
+export class DemoAppDataComponent implements OnInit {
+	public referenceList: ReferenceLink[];
+	public user?: StarkUser;
+	public time: string;
+
+	public constructor(
+		@Inject(STARK_USER_SERVICE) public userService: StarkUserService,
+		@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService,
+		@Inject(STARK_SESSION_SERVICE) public sessionService: StarkSessionService,
+		@Inject(STARK_APP_METADATA) public appMetadata: StarkApplicationMetadata
+	) {}
+
+	public ngOnInit(): void {
+		this.logger.debug(componentName + ": controller initialized");
+
+		this.referenceList = [
+			{
+				label: "Stark App Data component",
+				url: ""
```

Comment:
You can see what the url is if you run `npm run docs:serve` in the stark-ui package